common/utils/tests/Test_UtilsCudasort.cu

```diff
@@ -0,0 +1,228 @@
+/*
+* Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#include "gtest/gtest.h"
+
+#include <algorithm>
+#include <numeric>
+#include <random>
+#include <vector>
+
+#include <thrust/device_vector.h>
+#include <thrust/host_vector.h>
+
+#include <claragenomics/utils/cudasort.cuh>
+
+namespace claragenomics
+{
+namespace cudautils
```

Comment:
Optional: I would also remove the `cudautils` namespace. The user of the tested function will not be in the `cudautils` namespace.

Reply:
Make sense.
Done